projects/exui-common-lib/src/lib/services/cookie/cookie.service.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { CookieService } from './cookie.service';
+
+describe('CookieService', () => {
+  const mockDocument: any = {
+    cookie: ''
+  };
```

Comment:
In response to the question about another way to mock the behaviour of the document on the browser:

```
class MockDocument {
  private _cookie: string;
  constructor() {
    this._cookie = '';
  }
  public set cookie(val: string) {
    this._cookie += `; ${val}`;
  }
  public get cookie(): string {
    return this._cookie;
  }
}
```

And then:
const mockDocument: MockDocument = new MockDocument();

Reply:
Yeah, except it won't replace existing keys so it wouldn't work exactly as you'd expect. You'd probably want to use a dictionary instead... but it's just a mock!